r-package/grf/pkgdown/extra.js

```diff
@@ -0,0 +1,7 @@
+// From https://github.com/microsoft/LightGBM/blob/master/R-package/pkgdown/extra.js
```

Comment:
It'd be good to add short description of the issue that made this file necessary and link to the pkgdown github issue where you found the fix.